mypy/newsemanal/semanal.py

```diff
@@ -4041,6 +4041,11 @@ def add_imported_symbol(self, name: str, node: SymbolTableNode, context: Context
     def add_unknown_symbol(self, name: str, context: Context, is_import: bool = False,
                            target_name: Optional[str] = None) -> None:
         """Add symbol that we don't know what it points to (due to error, for example)."""
+        if is_import:
```

Comment:
Looks like `is_import` is always true. Maybe remove the argument (can happen in a separate PR)?

Reply:
OK, I will make a separate PR.